packages/story-editor/src/components/canvas/frameElement.js

```diff
@@ -264,18 +263,20 @@ function FrameElement({ id }) {
         data-element-id={id}
         {...box}
         tabIndex={-1}
+        role="presentation"
```

Comment:
💭  Not sure what role to put here

Reply:
I think `presentation` is fine here, but the big problem right now is that all elements (except text) don't have helpful text read out by screen readers.

Right now I'm hearing the text about "exiting the canvas area" but nothing about the element I'm focused on. We should be able to add one on each element type

Reply:
@samwhale would it be better to move this `speak(FRAME_ELEMENT_MESSAGE);` to fire when we press `enter` on the canvas container vs `onFocus` for the each element?

Reply:
I like that idea. It does get repetitive when going to each element.

Reply:
Dope. @BrookeGraham I think this is something we can take care of in a later ticket once we get everything in and have a sec to tie it all together.

Reply:
Added to the epic as a todo for later!